src/ucp/tag/offload.c

```diff
@@ -403,6 +403,30 @@ static ucs_status_t ucp_tag_offload_eager_zcopy(uct_pending_req_t *self)
                                     ucp_proto_am_zcopy_req_complete);
 }
 
+static size_t ucp_tag_offload_rndv_pack_rkey(ucp_request_t *sreq, ucp_lane_index_t lane,
+                                             void *rkey_buf, uint16_t *flags)
+{
+    ucp_ep_h ep = sreq->send.ep;
+    ucs_status_t status;
+
+    ucs_assert(UCP_DT_IS_CONTIG(sreq->send.datatype));
+
+    /* Check if the sender needs to register the send buffer -
+     * is its datatype contiguous and does the receive side need it */
+    if (ucp_ep_rndv_md_flags(ep, 0) & UCT_MD_FLAG_NEED_RKEY) {
```

Comment:
why do you pass `0` to `ucp_ep_rndv_md_flags`? Seems like this comes from `ucp_tag_rndv_pack_rkey`

Reply:
will be fixed